src/Profile/ProfileSourceLocalFs.php

```diff
@@ -33,7 +33,9 @@ public function getList()
 
     $list = [];
     foreach ($finder as $file) {
-      $filename = $file->getRealPath();
+      if (FALSE === $filename = $file->getRealPath()) {
+        $filename = $file->getPathname();
```

Comment:
Does getPathname work outside of a phar context? If so, perhaps we just use that instead rather than having conditional behaviour?

Reply:
I think it should (just tested cases, looks good) - given that the only place it's used is really in the context of extracting the filename.
I thought I may have been missing some secret sauce, hence the fallback.

Changed :)